recipe/meta.yaml

```diff
@@ -31,7 +31,11 @@ outputs:
         - libgcc-ng
       run_exports:
         strong:
+{% if cuda_major_minor < (11, 2) %}
```

Comment:
I am probably too tired, but shouldn't this be `(11, 2) <= cuda_major_minor < (12, 0)`?

Reply:
Ah had the cases flipped. Should be fixed now. Thanks! 😄